contributed_definitions/NXreflections.nxdl.xml

```diff
@@ -0,0 +1,415 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet type="text/xsl" href="nxdlformat.xsl" ?>
+<!--
+  # NXreflections: candidate NeXus definition for reflection data
+-->
+<definition type="group" name="NXreflections" category="contributed" extends="NXobject"
+    xmlns="http://definition.nexusformat.org/nxdl/@NXDL_RELEASE@"
```

Comment:
instead of @NXDL_RELEASE@, use 3.1, such as:

	    xmlns="http://definition.nexusformat.org/nxdl/3.1"
	    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
	    xsi:schemaLocation="http://definition.nexusformat.org/nxdl/3.1 ../nxdl.xsd"